src/lib/core/say.c

```diff
@@ -1209,7 +1210,10 @@ log_vsay(struct log *log, int level, const char *filename, int line,
 	 const char *error, const char *format, va_list ap)
 {
 	int errsv = errno;
+	if (flightrec_log(level, filename, line, error, format, ap) != 0)
+		fprintf(stderr, "Failed to log entry to flight recorder!\n");
```

Comment:
`flightrec_log` should never fails, because it just writes data to memory. Anyway, it's pointless to write to `stderr` on error - you'd spam the log should anything go wrong. Please remove this. Also, this function shouldn't affect `errno`.

Reply:
Ok, fixed. Also forgot that we can't use va_list twice, so we have to copy them via va_copy once.